python/nano/test/pytorch/tests/test_inference_pipeline.py

```diff
@@ -24,7 +24,7 @@
 from torch.utils.data import DataLoader
 from torch.utils.data.sampler import SequentialSampler
 from bigdl.nano.pytorch import Trainer
-from bigdl.nano.pytorch.inference import Optimizer
+from bigdl.nano.pytorch.inference import InferenceOptimizer
```

Comment:
change to from bigdl.nano.pytorch import InferenceOptimizer

Reply:
done